### DISCUSSION TITLE
Playback speed affects on replay

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#21710] [Q&A] [answered]
With different playback speed I have different play results. What's wrong with this?
For example, with 2x speed I have FC on non-FC replay:
![изображение](https://user-images.githubusercontent.com/55319814/208420517-480eb622-ceac-425f-aa2d-55d2b34c41f5.png)
(first and 4-th replays are the same play)

**Comment by smoogipoo** [comment, marked as answer]
Likely https://github.com/ppy/osu/issues/19766
